test/ops/align.jl

```diff
@@ -161,3 +161,10 @@ end
     @test _eval(count_knots(n4)) == _expected_count_knots(b4)
     @test _eval(count_knots(n_boolean)) == _expected_count_knots(b_boolean)
 end
+
+@testset "skip" begin
+    @test _eval(skip(n1, 1)) == b1[2:end]
```

Comment:
We should also test:
* Behaviour when given constant values
* That the identity map works (e.g. `skip(n1, 1) === skip(n1, 1)`)
* That skipping an empty node should always be a no-op `skip(<empty>, 42) === <empty>`, possibly for a couple of empty node types. See `TimeDag.empty_node`.

(This last point _might_ want to be made a separate issue. Right now TimeDag doesn't have a light-weight empty node type; instead it cheats by using an empty block node, which adds some overhead. But probably isn't needed too often.)